app/src/main/res/values/strings.xml

```diff
@@ -70,6 +70,7 @@
         <b>Reddit:</b>
         https://www.reddit.com/r/SimpleMobileTools
     </string>
+    <string name="open_dialpad_by_default">Open the dialpad by default when the app opens</string>
```

Comment:
move new strings into the appropriate place, this one should be at Settings, under group_subsequent_calls